guides/v2.3/config-guide/cli/logging.md

```diff
@@ -67,34 +68,46 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
     bin/magento cache:flush
     ```
 
+## Cron logging
+
+With the release of version 2.3.1, Magento now creates a separate `cron` log. 
+Magento recently made cron logging more verbose, which provided more information but lengthened the `system.log` considerably. 
+Moving `cron` info to a dedicated log makes both logs easier to read.
+
+By default, Magento writes `cron` info to `<install_directory>/var/log/cron.log`.
+
 ## Syslog logging
 
 By default, Magento writes _syslog_ logs to the operating system `syslog` file.
+As of Magento 2.3.1, you must use the `magento` command to enable or disable the syslog.
+The setting in the Magento Admin has been removed.
 
 #### To enable syslog logging:
 
-1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `1`.
+Logging to `syslog` is always enabled for developer and default modes, but disabled for production mode; even on Cloud where only production mode is supported.
+
+1. Use the `setup:config:set` command to change the `dev/syslog/syslog_logging` database value to `true`.
```

Comment:
Use the `setup:config:set --enable-syslog-logging=true|false` command to change the behavior